src/Cli/ProjectCommands.php

```diff
@@ -194,6 +210,43 @@ public function projectDerivativePull($remote, $options = ['as' => 'default', 'p
         }
     }
 
+    /**
+     * Performs minor updates for the WordPress Site Network repo.
+     *
+     * @param WorkingCopy $project_working_copy
+     *   The remote repo being worked on.
+     *
+     * @return void
+     */
+    protected function wordpressNetworkPreTagUpdates(WorkingCopy $project_working_copy)
+    {
+        $patch_path = realpath(__DIR__ . '/../../templates/wordpress-network/wp-config-pantheon.patch');
+
+        $project_working_copy->apply($patch_path);
+        $project_working_copy->add("wp-config-pantheon.php");
+        $project_working_copy->commit("Adds Multisite specific configuration to repo.");
+    }
+
+    /**
+     * Performs minor updates for the WordPress Site Network repo.
+     *
+     * @param WorkingCopy $project_working_copy
+     *   The remote repo being worked on.
+     *
+     * @return void
+     */
+    protected function wordpressNetworkPostTagUpdates(WorkingCopy $project_working_copy, $version)
+    {
+        $project_working_copy->tagDelete($version);
```

Comment:
If this code only runs on **new** tags, then this line should not be needed at all. This PR is much less alarming without this line, and could be re-opened if that was your preference. I'd still strongly prefer to see my recommended alternative implementation (in the WordPress upstream) though.